### PR TITLE
[13.0][ADD] partner_exception

### PR DESCRIPTION
**Pull request by JaumeBforgeFlow**
Applies base_exception module for contacts.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
Please rebase

**Review by MiquelRForgeFlow — APPROVED** [review]
Errors in CI are totally unrelated to this module.

File: partner_exception/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner Exception",
+    "summary": "Custom exceptions on partner contact",
+    "version": "13.0.1.2.0",
```
Comment (MiquelRForgeFlow) [inline partner_exception/__manifest__.py]:
```suggestion
    "version": "13.0.1.0.0",
```

**Review by JordiMForgeFlow — APPROVED** [review]

**Pull request merged**